### PR TITLE
Adds back making deswords + new item, double bladed energy cutlass

### PR DESCRIPTION
**Pull request by Fikou**
## About The Pull Request

this pr adds back making deswords from two eswords, while also improving a lot of the sprites related to eswords and adds a double energy cutlass (desword is removed from uplink, esword is limited to one stock if less than 25 pop in the uplink, also allows for more things like this with the new proc)

## Why It's Good For The Game

adds a new depth to pirates, where they can split up on roles for a melee man, gun man and inbetween, fixes sprites, adds back the "You're cool" message :-D .

## Changelog
:cl: Fikou
add: added making double energy swords, use an esword on another esword, you can deconstruct them back with a wrench
add: added the double energy cutlass, made with two energy cutlasses
del: removed the double energy sword from the uplink
imageadd: added new sprites for the double energy cutlass
/:cl:

## h
also i think i might have did some fucky wucky because one of my inhand files shows that it removes bytes

### CODE REVIEW

**Comment by plapatin** [comment]
now i can be darth tyranus AND darth maul at the same time

**Comment by Fikou** [comment]
what do you MEAN duplicate definition they are stating 2 different things someone hlep me fix this

**Comment by wesoda25** [comment]
I thought we made desword non craftable for balance reasons.

**Comment by Fikou** [comment]
> I thought we made desword non craftable for balance reasons.

i mean this is really kind of a nerf because you can get a 50% discount on desword while with this you have to pay at least 9 tc instead of 8 i dont really see the balance reasons

**Comment by wesoda25** [comment]
Yeah I guess that makes sense, discounts can go either way so pricing wouldn't really be a reason.

**Comment by kriskog** [comment]
Can I make an esword/ecutlass frankenstein and why not?

**Comment by Fikou** [comment]
> Can I make an esword/ecutlass frankenstein and why not?

because umm they have not connectable bottoms

**Comment by MrDoomBringer** [comment]
>use an esword on another esword, you can deconstruct them back with a wrench

Is this written in the desword's examine description?

**Comment by zxaber** [comment]
Tying the ability to combine items with player count sounds pretty arbitrary, to be honest.

**Comment by ExcessiveUseOfCobblestone** [comment]
the desword was made uncraftable because no one wants the discussion of "BUT WHAT ABOUT DESWORDS/SINGLE SWORDS" when touching literally anything relevant to either.

**Comment by ATH1909** [comment]
> the desword was made uncraftable because no one wants the discussion of "BUT WHAT ABOUT DESWORDS/SINGLE SWORDS" when touching literally anything relevant to either.

But it's still unavailable in lowpop rounds; you can't craft it when you're below the playercount that was needed to make a desword before this PR.

File: code/game/objects/items/melee/energy.dm [hunk 1]
```diff
@@ -178,7 +179,21 @@
...
+				if(ishuman(user))
+					user.adjustBrainLoss(25)
+			else
+				to_chat(user, "<span class='notice'>You attach the ends of the two energy swords, making a single double-bladed weapon! You're cool.</span>")
+				var/obj/item/twohanded/dualsaber/newSaber = new(user.loc)
+				qdel(W)
+				qdel(src)
+				user.put_in_hands(newSaber)
+		else
+			to_chat(user, "<span class='notice'>You can't seem to connect the blades together... maybe there's too little people?</span>")
```
Comment (JJRcop) [inline code/game/objects/items/melee/energy.dm]: This just seems cheap

**Comment by anconfuzedrock** [comment]
slightly less arbitrary would be adding a desword coupler that costs 1 tc and is only available at 25 people

sure it'd make deswords cost 1 more good boy point but who gives a fuck it's the most bland murderbone sword ever

File: code/game/objects/items/melee/energy.dm (hunk 1 above)
Comment (ExcessiveUseOfCobblestone) [reply code/game/objects/items/melee/energy.dm]: the wall just got 10 feet higher

**Comment by ExcessiveUseOfCobblestone** [comment]
> But it's still unavailable in lowpop rounds; you can't craft it when you're below the playercount that was needed to make a desword before this PR.

In a if not worse fashion.

**Comment by zxaber** [comment]
Requiring a connector would at least mean that you'd _know_ about the population issue before wasting eight TC on the second sword.

File: code/game/objects/items/melee/energy.dm (hunk 1 above)
Comment (PKPenguin321) [reply code/game/objects/items/melee/energy.dm]: Do a population check with the limited stock system and limit the stock to 1 esword should there be too few players

**Comment by MrPerson** [comment]
Adding all this special crafting/buying logic to reintroduce double esword crafting seems like a lot of work and snowflake special casing for literally no upside.

**Comment by Tlaltecuhtli** [comment]
dswords are super weak now that bolas exist

**Comment by ATH1909** [comment]
> slightly less arbitrary would be adding a desword coupler that costs 1 tc and is only available at 25 people
> 
> sure it'd make deswords cost 1 more good boy point but who gives a fuck it's the most bland murderbone sword ever

Maybe we could make it cost 0 TC, but have only one desword coupler available in each uplink?

**Comment by Fikou** [comment]
ok now the esword stock is based on pop in the uplink

File: code/game/objects/items/melee/energy.dm (hunk 1 above)
Comment (Fikou) [reply code/game/objects/items/melee/energy.dm]: ok i say this is resolved ok

**Comment by optimumtact** [comment]
hhahahahah boss I did it again, I opened a pr for a feature that was removed

epic, that will show them he chuckled to himself, but then, he saw it, a frown crossed his face.

CLOSED? AN OUTRAGE, HE SPUTTERED, SURELY HIS LOGICAL ARGUMENTS HAD CHANGED THE VIEW OF THE FEATURE, HE FELL BACK INTO HIS CHAIR, DRAINED. WHy COULD THEY HAVE NOT UNDERSTOOD HIS GENIUS?

**Comment by Fikou** [comment]
> hhahahahah boss I did it again, I opened a pr for a feature that was removed
> 
> epic, that will show them he chuckled to himself, but then, he saw it, a frown crossed his face.
> 
> CLOSED? AN OUTRAGE, HE SPUTTERED, SURELY HIS LOGICAL ARGUMENTS HAD CHANGED THE VIEW OF THE FEATURE, HE FELL BACK INTO HIS CHAIR, DRAINED. WHy COULD THEY HAVE NOT UNDERSTOOD HIS GENIUS?

ok

**Comment by MrDoomBringer** [comment]
orange man u good?

**Comment by ExcessiveUseOfCobblestone** [comment]
>>>>>>>>>>>>CLOSED? AN OUTRAGE, HE SPUTTERED, SURELY HIS LOGICAL ARGUMENTS HAD CHANGED THE VIEW OF THE FEATURE, HE FELL BACK INTO HIS CHAIR, DRAINED. WHy COULD THEY HAVE NOT UNDERSTOOD HIS GENIUS?

**Comment by FantasticFwoosh** [comment]
That's one for my cringe compilation :camera_flash:

**Comment by PKPenguin321** [comment]
😬 <-- cringe react

**Pull request closed without merging**